hashtheplanet/sql/db_connector.py

```diff
@@ -144,3 +142,11 @@ def get_all_hashs(session):
         stmt = select(Hash)
         hashs = session.execute(stmt).scalars().all()
         return hashs
+
+    @staticmethod
+    def get_static_files(session):
+        """
+        Returns all files ending with .txt or .html
+        """
+        return list(map(lambda x: x.path, (session.query(File).filter(File.path.like('%.txt')).all() + \
+            session.query(File).filter(File.path.like('%.html')).all())))
```

Comment:
Here, you could use only one request (either by using a 'or' statement or by using a more complex regex) and you can select only the field path so you don't need to use a map.

Reply:
I changed the two `like` with a regex, but I cannot remove the `map` function because the sql request returns a tuple containing the column `path`: `("foobar.txt,)`